example/runtime/pytorch-e2e/requirements.txt

```diff
@@ -1,5 +1,7 @@
 # for pfp
 pycocotools
+# for api decorator
+gradio
```

Comment:
Will installing the extra pkg of starwhale in the `starwhale` base image be better?

Reply:
updated